### PR TITLE
java coverage

### PR DESCRIPTION
**Pull request by sanwan**

### CODE REVIEW

File: sdk/java/pom.xml [hunk 1]
```diff
@@ -9,14 +9,15 @@
...
+		<argLine>-Djdk.net.URLClassPath.disableClassPathURLCheck=true -Djava.library.path=${project.basedir}/../mount/libjfs:${java.library.path}</argLine>
 	</properties>
 	<build>
 		<plugins>
 			<plugin>
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>2.19.1</version>
 				<configuration>
-					<argLine>-Djdk.net.URLClassPath.disableClassPathURLCheck=true -Djava.library.path=${project.basedir}/../mount/libjfs:${java.library.path}</argLine>
+					<argLine>@{argLine}</argLine>
```
Comment (davies) [inline sdk/java/pom.xml:20]: ?

**Comment by davies** [comment]
@sanwan I can't see which part of the source code is covered by existing tests, how?

File: sdk/java/pom.xml (hunk 1 above)
Comment (sanwan) [reply sdk/java/pom.xml:20]: If surefire and jacoco are used in same pom file , argline paramters of jacoco will be overwritted .More details may refer to https://stackoverflow.com/questions/18107375/getting-skipping-jacoco-execution-due-to-missing-execution-data-file-upon-exec.

Comment (davies) [reply sdk/java/pom.xml:20]: I see, it's better to leave a comment here.

**Comment by davies** [comment]
I can see the coverage of some of the Java files, the most important one (JuiceFileSystemImpl.java) is loaded in another classloader, how can we measure the coverage of it?

File: sdk/java/pom.xml (hunk 1 above)
Comment (sanwan) [reply sdk/java/pom.xml:20]: ok

**Pull request merged**